patterns/vsi-extension/variables.tf

```diff
@@ -51,63 +50,53 @@ variable "access_tags" {
 }
 
 variable "image_name" {
-  description = "Image ID used for the VSI. Run the 'ibmcloud is images' CLI command to find available images. The IDs are different in each region."
+  description = "The image ID used for the VSI. You can run the `ibmcloud is images` CLI command to find available images. The IDs are different in each region."
   type        = string
-  default     = "ibm-ubuntu-22-04-2-minimal-amd64-1"
+  default     = "ibm-ubuntu-22-04-3-minimal-amd64-1"
 }
 
-variable "machine_type" {
-  description = "VSI machine type"
+variable "vsi_instance_profile" {
+  description = "The VSI image profile. You can run the `ibmcloud is instance-profiles` CLI command to see available image profiles."
   type        = string
-  default     = "cx2-2x4"
+  default     = "cx2-4x8"
 }
 
 variable "user_data" {
-  description = "User data to initialize VSI deployment."
+  description = "The user data to transfer to the instance. [Learn more](https://cloud.ibm.com/docs/vpc?topic=vpc-user-data)."
   type        = string
   default     = null
 }
 
 variable "boot_volume_encryption_key" {
-  description = "The CRN of the boot volume encryption key."
+  description = "The CRN of the boot volume encryption key. [Learn more](https://cloud.ibm.com/docs-draft/secure-infrastructure-vpc?topic=secure-infrastructure-vpc-ext-with-vsi)."
   type        = string
 }
 
-variable "existing_kms_instance_guid" {
-  description = "The GUID of the KMS instance that holds the key specified in `var.boot_volume_encryption_key`."
-  type        = string
-}
-
-variable "skip_iam_authorization_policy" {
-  type        = bool
-  description = "By default (true), the Landing Zone VPC creates an IAM authorization policy that permits all storage blocks to read the encryption key from the KMS instance. Set to false to create the authorization policy in a different KMS instance, and specify the GUID of the KMS instance in the existing_kms_instance_guid variable."
-  default     = true
-}
-
 variable "vsi_per_subnet" {
-  description = "The number of VSI instances for each subnet."
+  description = "The number of virtual servers to create on each VSI subnet."
   type        = number
   default     = 1
 }
 
 variable "subnet_names" {
-  description = "The subnets to deploy the VSI instances to."
+  description = "The subnets where you want to deploy the VSI. If not specified, the VSI is deployed to all the subnets in the VPC. [Learn more](https://cloud.ibm.com/docs-draft/secure-infrastructure-vpc?topic=secure-infrastructure-vpc-ext-with-vsi)."
```

Comment:
Perhaps reword first sentence to "A list of subnet names where you want to deploy a VSI."